### PR TITLE
Template

### PR DESCRIPTION
**Pull request by dmitrybublik**

### CODE REVIEW

**Comment by godrose** [comment]
@dmitrybublik Please add example of usage

File: logofx_client_bootstrapping_samples/.template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "classifications": [ "LogoFX", "Wpf" ], 
+  "name": "LogoFX.Client.Bootstrapping.Samples",
+  "identity": "LogoFX.Client.Bootstrapping.Samples.CSharp",         
+  "groupIdentity":"Samples.Bootstrapping.Template",
+  "shortName": "logofx-client-bootstrapping",                    // You can create the project using this short name instead of the one above.
+  "tags": {
+    "language": "C#",                         // Specify that this template is in C#.
+    "type":"project"
+  },
+  "sourceName": "logofx_client_bootstrapping_samples",              // Will replace the string 'logofx_client_bootstrapping_samples' with the value provided via -n.
```
Comment (godrose) [inline logofx_client_bootstrapping_samples/.template.config/template.json]: check the namespaces issue

File: logofx_client_bootstrapping_samples/App.xaml [hunk 2]
```diff
@@ -1,4 +1,3 @@
-﻿<Application x:Class="LogoFX.Client.Bootstrapping.Samples.WPF.App"
+﻿<Application x:Class="logofx_client_bootstrapping_samples.App"
```
Comment (godrose) [inline logofx_client_bootstrapping_samples/App.xaml]: Check namespace

File: logofx_client_bootstrapping_samples/App.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using Solid.Core;
+
+namespace logofx_client_bootstrapping_samples
```
Comment (godrose) [inline logofx_client_bootstrapping_samples/App.xaml.cs]: check namespaces

File: logofx_client_bootstrapping_samples/AppBootstrapper.cs [hunk 4]
```diff
@@ -1,8 +1,9 @@
-﻿using LogoFX.Client.Bootstrapping.Adapters.SimpleContainer;
-using LogoFX.Client.Bootstrapping.Samples.WPF.ViewModels;
+﻿using LogoFX.Client.Bootstrapping;
+using LogoFX.Client.Bootstrapping.Adapters.SimpleContainer;
+using logofx_client_bootstrapping_samples.ViewModels;
 using Solid.Practices.Composition;
 
-namespace LogoFX.Client.Bootstrapping.Samples.WPF
+namespace logofx_client_bootstrapping_samples
```
Comment (godrose) [inline logofx_client_bootstrapping_samples/AppBootstrapper.cs]: check namespace

File: logofx_client_bootstrapping_samples/AppBootstrapper.cs [hunk 5]
```diff
@@ -14,7 +15,7 @@ public AppBootstrapper()
 
         public override CompositionOptions CompositionOptions => new CompositionOptions
         {
-            Prefixes = new[] {"LogoFX.Client.Bootstrapping.Samples"}
+            Prefixes = new[] { "logofx_client_bootstrapping_samples" }
```
Comment (godrose) [inline logofx_client_bootstrapping_samples/AppBootstrapper.cs]: check namespace

**Review by glyad — APPROVED** [review]

File: LogoFX.Samples.Client.Bootstrapping/.template.config/template.json [hunk 6]
```diff
@@ -2,14 +2,14 @@
...
+  "name": "LogoFX.Samples.Client.Bootstrapping",
+  "identity": "LogoFX.Samples.Client.Bootstrapping.CSharp",         
   "groupIdentity":"Samples.Bootstrapping.Template",
   "shortName": "logofx-client-bootstrapping",                    // You can create the project using this short name instead of the one above.
   "tags": {
     "language": "C#",                         // Specify that this template is in C#.
     "type":"project"
   },
-  "sourceName": "logofx_client_bootstrapping_samples",              // Will replace the string 'logofx_client_bootstrapping_samples' with the value provided via -n.
+  "sourceName": "LogoFX.Samples.Client.Bootstrapping",              // Will replace the string 'LogoFX.Client.Bootstrapping.Samples' with the value provided via -n.
```
Comment (godrose) [inline LogoFX.Samples.Client.Bootstrapping/.template.config/template.json]: Shouldn't it be "LogoFX.Samples.Client.Bootstrapping" in all places?
It looks like we still have LogoFX.Client.Samples somewhere

**Review by godrose — CHANGES_REQUESTED** [review]

**Review by godrose — APPROVED** [review]

**Pull request merged**